Makefile

```diff
@@ -108,4 +108,6 @@ docker:
 	docker exec tmp-make-cluster-test sh -c "ipfs-cluster-service -v"
 	docker kill tmp-make-cluster-test
 
+prcheck: deps check service ctl test intall test_sharness
```

Comment:
type: `intall`. But a test target should not modify the running system. Perhaps safest to run without `test_sharness`.

Reply:
What a horrible typo! :see_no_evil: 
Let me change this a little later today. Will take a look at #539 as well while on it.

Reply:
> But a test target should not modify the running system.

I think most developers tend to use a separate $GOPATH for development. I personally use a different $GOPATH for every project. And during development anyway developers would be installing the modified version. I think we can keep `install` and `test_sharness`

Reply:
Don't assume what people do. I don't do any of those things. And your GOBIN path can be set to anything even if you're on a different GOPATH. Only `make install` should overwrite anything outside the project folder.

Reply:
Made those changes https://github.com/ipfs/ipfs-cluster/pull/542/commits/7d9c12dd9a34ca467a80f6a7e9513cd954592efa